src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,26 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File file = new File(fromFileName);
+        try {
+            List<String> strings = Files.readAllLines(file.toPath());
+            System.out.println(strings);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read fails ", e);
```

Comment:
The error message 'Can't read fails' seems to contain a typo. Consider changing it to 'Can't read file'.